bokeh/charts/_charts.py

```diff
@@ -144,17 +147,28 @@ def start_plot(self, xgrid, ygrid):
         if ygrid:
             self.make_grid(1, yaxis.ticker)
 
-        # Add tools
+        # Add tools if supposed to
         if self.tools:
+            # only create pan and zoom tools if chart is not categorical...
+            if self.categorical:
+                tools_to_remove = ['pan', 'zoom']
+            else:
+                tools_to_remove = []
```

Comment:
I also wonder if maybe `_add_tools_to_plot` shouldn't just check for categorical ranges in one place. Taking `tools_to_remove` is more general, but I can't think of any other use scenarios for it, and some day cat ranges will work with pan/zoom so then there would just be one place to fix up.